src/platform/forms-system/src/js/components/FormSignature.jsx

```diff
@@ -137,37 +141,35 @@ export const FormSignature = ({
 
 FormSignature.propTypes = {
```

Comment:
Alphabetized proptypes and disabled a couple prop order rules that couldn't be enforced (i.e., `onSectionComplete` was supposed to be at the top of the list AND at the end..)